wren-launcher/commands/launch.go

```diff
@@ -170,9 +170,24 @@ func Launch() {
 			panic("Timeout")
 		}
 
-		// check if WrenAI is started
-		err = utils.CheckWrenAIStarted(url)
+		// check if ui is ready
+		err := utils.CheckWrenUIStarted(url)
```

Comment:
nit: `CheckUIServiceStarted`

Reply:
delivered in 38544ea2cca6541252f3c7c738bd28b0d0526e93